src/Cake.Common/Tools/TextTransform/TextTransformAliases.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.TextTransform
+{
+    /// <summary>
+    /// Contains functionality related to transforming templates
+    /// </summary>
+    [CakeAliasCategory("TextTransform")]
+    public static class TextTransformAliases
+    {
+        /// <summary>
+        /// Transform a text template.
+        /// </summary>
+        /// <example>
+        /// <code>
+        /// // Transform a .tt template.
+        /// var transform = File("./src/Cake/Transform.tt")
+        /// TransformTemplate(transform, new TextTransformSettings { OutputFile="./src/Cake/Transform.cs" });
```

Comment:
@gep13 Verified agains code in this PR
```cake
#tool Mono.TextTransform
var transform = File("Transform.tt");
TransformTemplate(transform, new TextTransformSettings {});
```
on a `Transform.tt` that looks like this
```
<#@ output extension=".sql" #>
use Master
GO
```
generates an `Transform.sql` that looks like this
```sql
use Master
GO
```

So @gep13 your feedback is actually valid as I think @RLittlesII should add an alias overload that doesn't require an settings parameter.